syntaxes/omt.tmLanguage.json

```diff
@@ -343,11 +343,11 @@
                 }
             ]
         },
-        "_notification-action-def": {
+        "_system-message-action-def": {
             "patterns": [
                 { "include": "#comments" },
                 {
-                    "match": "(?:\\G|^) +(icon|id): *(\\w*)(.*)$",
+                    "match": "(?:\\G|^) +(id|uuid): *(\\w*)(.*)$",
```

Comment:
`icon` and `id` are still the unquoted string type properties available, `uuid` should be added to the query type properties on line 370.